runtime/queries/markdown_inline/highlights.scm

```diff
@@ -16,10 +16,10 @@
 ] @string.escape
 
 ; Conceal codeblock and text style markers
-((code_span_delimiter) @markup.raw.delimiter
-  (#set! conceal ""))
-
-((emphasis_delimiter) @conceal
+([
+  (code_span_delimiter)
+  (emphasis_delimiter)
+] @conceal
```

Comment:
`@conceal` isn’t listed as a standard capture (unless I missed it). Is there some kind of exception in this case?

Reply:
Yes; it's a "special capture" like `@spell` (or, rather, like `@_foo`). See `:h treesitter-highlight-conceal`.